src/main/java/seedu/address/model/issue/Issue.java

```diff
@@ -71,6 +71,12 @@ public Status getStatus() {
         return this.status;
     }
 
+    @Override
+    public String toString() {
```

Comment:
May want to create a separate method for the display implementation, since the `toString()` method is used to store the `IssueId` as JSON in the constructor of `JsonAdaptedIssue`